README.md

```diff
@@ -0,0 +1,4 @@
+# Belka-datalog
+
+Belka-datalog is a flavor of datalog query language, implemented for a deductive database.
```

Comment:
We are quite far from a deductive database yet.

Reply:
sam takoy!